sql/2020/07_SEO/pages_wpt_bodies_by_device_and_title_char_count.sql

```diff
@@ -0,0 +1,41 @@
+#standardSQL
+# page wpt_bodies metrics grouped by device and title char count
+
+# helper to create percent fields
+CREATE TEMP FUNCTION AS_PERCENT (freq FLOAT64, total FLOAT64) RETURNS FLOAT64 AS (
+  ROUND(SAFE_DIVIDE(freq, total), 4)
+);
+
+# returns all the data we need from _wpt_bodies
+CREATE TEMPORARY FUNCTION get_wpt_bodies_info(wpt_bodies_string STRING)
+RETURNS STRUCT<
+    title_char_count INT64
+> LANGUAGE js AS '''
+var result = {};
+try {
+    var wpt_bodies = JSON.parse(wpt_bodies_string); 
+
+    if (Array.isArray(wpt_bodies) || typeof wpt_bodies != 'object') return result;
+
+    if (wpt_bodies.title && wpt_bodies.title.rendered && wpt_bodies.title.rendered.primary) {
+      result.title_char_count = wpt_bodies.title.rendered.primary.characters;
+    }
+
+} catch (e) {}
+return result;
+''';
+
+SELECT
+client,
+COUNT(0) AS total, 
+wpt_bodies_info.title_char_count as char_count
```

Comment:
Ditto on percentiles

Reply:
Already in pages_wpt_bodies_by_device_and_percentile.sql. Removed this one.